CHANGELOG.txt

```diff
@@ -6,6 +6,46 @@
 *                                                *
 **************************************************
 
+<<<<<<< HEAD
```

Comment:
Merge conflict...you need to be reading what the terminal output that:

```
git pull <remote> <branch name>
```

Gives you and then acting on that before pushing to your remote.

Reply:
the conflict is resolved..